uber/automated_emails.py

```diff
@@ -330,7 +332,7 @@ def __init__(self, conf):
 IT WILL CAUSE ANY EMAILS THAT HAVE ALREADY SENT TO RE-SEND.
 ************************************************************************
 
-WORK WE NEED TO DO: Going forward, every single email category below should have an explicit 'ident' set. However,
```

Comment:
This docstring can be either removed wholesale or replaced with a shorter comment explaining the reason behind having `ident` in the first place. It shouldn't be left like this because it still describes old behavior (e.g., on line 328 above, it still says `If an 'ident' is not set (not recommended), 'ident' will default to be the 'subject' line.`).